docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "069a3ce3",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# %%capture \n",
+    "# !pip uninstall cleanlab -y \n",
+    "# !pip install git+https://github.com/ericwang1997/cleanlab.git@token_classification \n",
+    "# !pip install termcolor "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "33d67688",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# package versions: \n",
+    "# cleanlab==2.0.0 (ericwang/cleanlab -b token_classification)\n",
+    "# termcolor==1.1.0 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Token Classification Label Error Detection - Part 2 \n",
+    "\n",
+    "In this tutorial, we show how you can use cleanlab to find potential label errors in a NLP token classification dataset. Here, we use CONLL-2003 which contains 20,718 sentences and 301,361 tokens. In a standard NLP token-classification task, the model predicts the entity of each token within each sentence. In this example, the named entity includes miscellaneous `MISC`, location `LOC`, person `PER`, and organization `ORG`. Unnamed entities are labeled as `O`. \n",
```

Comment:
"in a NLP token classification dataset" ->
"in text datasets used for the NLP task of token classification."